pkgs/development/python-modules/hiyapyco/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, pyyaml, jinja2  }:
+
+buildPythonPackage rec {
+  pname = "HiYaPyCo";
+  version = "0.4.16";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1qb7fb90jzpzx84adzwsabxq4hb5nc44szxn08nxkr2flzp1kvpz";
+  };
+
+  propagatedBuildInputs = [ pyyaml jinja2 ];
+
+  meta = with lib; {
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Reply:
Good point. Apparently, the Pypi version does not include all the dependencies to run the test. I've switched to the source [from Github](https://github.com/zerwes/hiyapyco) and the tests are now executed. LGTM on Python 2 and 3.